<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helpOnLab</class>
 <widget class="QMainWindow" name="helpOnLab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>helpOnLab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>651</width>
      <height>441</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>651</width>
      <height>441</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Calculatrice de dilution</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>331</width>
        <height>301</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Calculer le Fd</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> Volume initial</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> Volume final</string>
        </property>
       </widget>
       <widget class="QPushButton" name="calculateButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>191</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Facteur de dilution :</string>
        </property>
       </widget>
       <widget class="QLabel" name="resultLabel">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>120</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>11</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>μL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>40</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>μL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="viInput">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="vfInput">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Volume du diluant :</string>
        </property>
       </widget>
       <widget class="QLabel" name="result2Label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/alertIcon.png&quot; width=&quot;50&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>En cours de developpement !!</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Calculer un volume</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Calculer mix Pcr</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>En cours de developpement</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Converssion</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuA_propos"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
